dspace-server-webapp/src/main/java/org/dspace/app/rest/security/WorkflowRestPermissionEvaluatorPlugin.java

```diff
@@ -62,7 +62,7 @@ public boolean hasDSpacePermission(Authentication authentication, Serializable t
         //This plugin currently only evaluates READ access
         DSpaceRestPermission restPermission = DSpaceRestPermission.convert(permission);
         if (!DSpaceRestPermission.READ.equals(restPermission)
-                || Constants.getTypeID(targetType) != Constants.WORKFLOWITEM) {
+                || !StringUtils.equals(targetType, "WORKFLOWITEM")) {
```

Comment:
Same as above, should still be a Constant, so that we aren't needing to hardcode the same string in multiple places

Reply:
Done.